src/api/transform/reasoning.ts

```diff
@@ -36,7 +34,7 @@ export const getOpenRouterReasoning = ({
 	shouldUseReasoningBudget({ model, settings })
 		? { max_tokens: reasoningBudget }
 		: shouldUseReasoningEffort({ model, settings })
-			? reasoningEffort !== "minimal"
+			? reasoningEffort && reasoningEffort !== "minimal"
```

Comment:
I'm pretty sure we can just pass in `{ effort: 'minimal' }` if I'm understanding things correctly.
```suggestion
			? reasoningEffort
```